src/wallet/rpcwallet.cpp

```diff
@@ -430,6 +439,8 @@ static UniValue sendtoaddress(const JSONRPCRequest& request)
             "       \"" + FeeModes("\"\n\"") + "\""},
                     {"avoid_reuse", RPCArg::Type::BOOL, /* default */ "true", "(only available if avoid_reuse wallet flag is set) Avoid spending from dirty addresses; addresses are considered\n"
             "                             dirty if they have previously been used in a transaction."},
+                    {"verbose", RPCArg::Type::BOOL, /* default */ "false",
+                            "Optional, the fee reason (only present when `verbose` is passed)"},
```

Comment:
This is just a tad bit confusing, I'd recommend something like "If true, return the fee reason"

Reply:
Er, something more generic would be "If true, return extra information about the transaction created."